### PR TITLE
28 information panel

### PR DESCRIPTION
**Pull request by rkzel**
Styling for information panel, no logic for now.

### CODE REVIEW

File: app/src/assets/IconEthereum.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            id="Shape"
+          />
+        </g>
+      </g>
+    </g>
+  </svg>
+)
+
```
Comment (Schwartz10) [inline app/src/assets/IconEthereum.js]: Could you add default propTypes for color like you did for `IconGitHub`?

File: app/src/assets/IconLocation.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <g id="noun_Location_1355745" fill={props.color} fillRule="nonzero">
+        <path
+          d="M14.5,0.164772727 C9.29273612,0.165037939 4.49512356,2.98904104 1.96753157,7.54171893 C-0.560060428,12.0943968 -0.420084153,17.6596898 2.33318182,22.0795455 L14.5,39.875 L26.6668182,22.0795455 C29.4200842,17.6596898 29.5600604,12.0943968 27.0324684,7.54171893 C24.5048764,2.98904104 19.7072639,0.165037939 14.5,0.164772727 Z M14.5,19.7727273 C11.5879531,19.7727273 9.22727273,17.4120469 9.22727273,14.5 C9.22727273,11.5879531 11.5879531,9.22727273 14.5,9.22727273 C17.4120469,9.22727273 19.7727273,11.5879531 19.7727273,14.5 C19.7727273,15.8984142 19.2172094,17.239553 18.2283812,18.2283812 C17.239553,19.2172094 15.8984142,19.7727273 14.5,19.7727273 Z"
+          id="Shape"
+        />
+      </g>
+    </g>
+  </svg>
+)
+
```
Comment (Schwartz10) [inline app/src/assets/IconLocation.js]: Could you add default propTypes for color like you did for `IconGitHub`?

File: app/src/assets/IconTwitter.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            id="Shape"
+          />
+        </g>
+      </g>
+    </g>
+  </svg>
+)
+
```
Comment (Schwartz10) [inline app/src/assets/IconTwitter.js]: Could you add default propTypes for color like you did for `IconGitHub`?

File: app/src/components/informationPanel/InformationCard.js [hunk 4]
```diff
@@ -1,91 +1,170 @@
...
-    if (!userLoaded) return ''
-    return boxes[ethereumAddress].publicProfile[field] || ''
-  }
+  const fields = boxes[ethereumAddress].publicProfile
+
+  fields.website = 'https://autark.xyz'
+  fields.github = 'rkzel'
+  fields.location = 'Mons Olympus'
+  fields.twitter = 'autarklabs'
+
```
Comment (Schwartz10) [inline app/src/components/informationPanel/InformationCard.js]: please remove these hardcoded values for now

File: app/src/components/informationPanel/InformationCard.js [hunk 5]
```diff
@@ -1,91 +1,170 @@
...
   const { boxes } = useContext(BoxContext)
   const { dispatchModal } = useContext(ModalContext)
 
   const userLoaded = !!boxes[ethereumAddress]
 
-  const getValue = field => {
-    if (!userLoaded) return ''
-    return boxes[ethereumAddress].publicProfile[field] || ''
-  }
+  const fields = boxes[ethereumAddress].publicProfile
```
Comment (Schwartz10) [inline app/src/components/informationPanel/InformationCard.js]: Let's remove this variable, and rename the properties that use `fields` to be `boxes[ethereumAddress].publicProfile[field]`

The reason for this is that if `!userLoaded`, `boxes[ethereumAddress]` will be undefined, so `boxes[ethereumAddress].publicProfile` will throw an error - cannot read property publicProfile of undefined.

File: app/src/components/informationPanel/InformationCard.js [hunk 6]
```diff
@@ -1,91 +1,170 @@
...
+  fields.twitter = 'autarklabs'
+
+  return !userLoaded ? (
+    <div>No profile</div>
+  ) : (
+    <StyledCard>
+      <Information>
+        <Details>
+          <Text.Block size="xxlarge" style={{ fontWeight: '700' }}>
+            {fields.name ? fields.name : 'name'}
```
Comment (Schwartz10) [inline app/src/components/informationPanel/InformationCard.js]: Should be `boxes[ethereumAddress].publicProfile.name` (see comment above)

File: app/src/components/informationPanel/InformationCard.js [hunk 7]
```diff
@@ -1,91 +1,170 @@
...
+          </Text.Block>
+          <Text.Block size="normal">
+            {fields.description ? fields.description : 'description'}
```
Comment (Schwartz10) [inline app/src/components/informationPanel/InformationCard.js]: here the same change as above ^^^ and for the rest of the `fields` below..

File: app/src/assets/IconEthereum.js (hunk 1 above)
Comment (rkzel) [reply app/src/assets/IconEthereum.js]: done.

File: app/src/assets/IconLocation.js (hunk 2 above)
Comment (rkzel) [reply app/src/assets/IconLocation.js]: done.

File: app/src/assets/IconTwitter.js (hunk 3 above)
Comment (rkzel) [reply app/src/assets/IconTwitter.js]: done.

File: app/src/components/informationPanel/InformationCard.js (hunk 4 above)
Comment (rkzel) [reply app/src/components/informationPanel/InformationCard.js]: removed.

File: app/src/components/informationPanel/InformationCard.js (hunk 5 above)
Comment (rkzel) [reply app/src/components/informationPanel/InformationCard.js]: improved this bit.

File: app/src/components/informationPanel/InformationCard.js (hunk 6 above)
Comment (rkzel) [reply app/src/components/informationPanel/InformationCard.js]: improvements made for the last comment should make this more resistant...

File: app/src/components/informationPanel/InformationCard.js (hunk 7 above)
Comment (rkzel) [reply app/src/components/informationPanel/InformationCard.js]: improvements made for the last comment should make this more resistant...

**Comment by Schwartz10** [comment]
@rkzel just assigned you to #17 - can we include that in this PR?

**Pull request merged**